src/utils/mappers.js

```diff
@@ -15,5 +15,9 @@ export function req ( path ) {
 }
 
 export function globalify ( name ) {
-	return 'global.' + name;
-}
\ No newline at end of file
+  	if ( /^__dep\d+__$/.test( name ) ) {
```

Comment:
Per @eventualbuddha 's comment I changed the UMD implementation to pass `undefined` instead of `global.__dep0__`. I did it in the least intrusive way, which is arguably hacky. A wider refactor may be better, but I lack a wider context on this codebase.